### PR TITLE
Fix bug

### PR DESCRIPTION
**Pull request by jerrywithaz**

### CODE REVIEW

File: congress/patch/patch.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+from functools import wraps
+
+# The patch module is loaded after the task module is loaded, so all task
+# modules are on the import path.
+from congress.tasks import utils, govinfo, bills, votes
+
+session = boto3.Session(profile_name="Democrasee")
+client = session.client("s3")
+transport_params = {"client": client}
+bucket = "democrasee-storage"
```
Comment (michaelblyons) [inline congress/patch/patch.py:16]: I'm sorry. Did you mean to hard code your S3 bucket name into the pull request?

**Review by michaelblyons — CHANGES_REQUESTED** [review]
Maybe include a description for what your problem was and what you're trying to do with this code?

**Comment by dwillis** [comment]
@jerrywithaz Can you describe what this PR is supposed to be fixing? It looks like you've added a file that includes information about your own boto/AWS profile. That shouldn't be a part of the project.

**Comment by jerrywithaz** [comment]
@dwillis @michaelblyons Damn, sorry guys, I did not mean to push this up lol

**Pull request closed without merging**